### PR TITLE
week 1: tweaks, and changes, a few more labs

### PR DESCRIPTION
**Pull request by Habenzy**

### CODE REVIEW

File: public/lessons/javascript/functions.md [hunk 1]
```diff
@@ -252,7 +251,184 @@ let add = (x,y) => x + y;
...
+ageCalc(age) // should print "You are 852055200 seconds old." to the console
+```
+
+How could we use ARGV to make this more modular?
+
+# Flip it around!
+
+Can you write the inverse function; one that takes a number of seconds and tells you the exact age?
+
+You can get the current date by calling `Date.now()` which will give you a time in milliseconds, and the date you were born by createing a new `Date` object. You can then figure out the time that's elapsed in milliseconds by subtracting the date you were born from the current date
```
Comment (ncastle) [inline public/lessons/javascript/functions.md:274]: typo in `creating` here

File: public/lessons/javascript/functions.md [hunk 2]
```diff
@@ -252,7 +251,184 @@ let add = (x,y) => x + y;
...
+
+<details>
+<summary>Solution</summary>
+<div>
+
+```js
+function supplyCalc(age, amountPerDay, item) {
+  let amountPerYear = amountPerDay * 365
+  let numberOfYears = 100 - age
+  let totalNeeded = amountperYear * numberOfYears
```
Comment (ncastle) [inline public/lessons/javascript/functions.md:363]: amountPerYear*

File: public/lessons/javascript/input_and_output.md [hunk 3]
```diff
@@ -150,9 +150,9 @@ What happens? Is this what you expected?
 * Fortunately, there's an easy fix
 * If you send the message `trim` to a string, it will remove all SPACES and NEWLINES from both ends
 
-# LAB: fixing Hello, Friend
+# Solution: fixing Hello, Friend
 
-* Change the program to look like this:
+* If you're still struggling to sanatize your input change the program to look like this:
```
Comment (ncastle) [inline public/lessons/javascript/input_and_output.md:155]: If you're still struggling to `sanitize` your input`,` change

File: public/lessons/javascript/input_and_output.md [hunk 4]
```diff
@@ -222,8 +224,6 @@ function ask(questionText) {
 }
 ```
 
-> This is called "boilerplate code" -- you don't need to fully understand it before using it.
```
Comment (ncastle) [inline public/lessons/javascript/input_and_output.md:225]: Is there somewhere else you explain boilerplate code?

File: public/lessons/javascript/logic.md [hunk 5]
```diff
@@ -169,7 +169,7 @@ You can make more complicated logical expressions using conjunctions:
 |Conjunction|Operator|Example|Meaning|
 |---|---|---|---|
 | AND | `&&` | `X && Y` | "are both X and Y true?" |
-| OR | <code>&#124;&#124;</code> | <code>X &#124;&#124; Y</code> | "is either X or Y (or both) true?" |
+| OR | `||` | `X || Y` | "is either X or Y (or both) true?" |
```
Comment (ncastle) [inline public/lessons/javascript/logic.md:172]: It looks like these aren't actually visible when the markdown renders

File: public/lessons/javascript/values.md [hunk 6]
```diff
@@ -33,9 +47,89 @@ A **string** is an object that's a collection of characters, like a word or a se
...
+    * `"titanic".toUpperCase()`
+    * `"QUIETLY".toLowerCase()`
+    * `"Java".repeat(10)`
+    * `"banana".length`
+    * `"berry".charAt(1)`
+    * `"berry".charAt(0)`
+    * `"apple"[3]`
+    * `"banana".includes("nan")`
+    * `"banana".endsWith("ana")`
+    * `"blueberry".replace("blue", "black")`
```
Comment (ncastle) [inline public/lessons/javascript/values.md:128]: The back ticks on each of these is rendered with the markdown. Is this intentional?

File: public/lessons/javascript/input_and_output.md (hunk 4 above)
Comment (Habenzy) [reply public/lessons/javascript/input_and_output.md:225]: Nope

**Pull request merged**